github/ci/prow-sig-scale-workloads/hack/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source $(dirname "$0")/common.sh
+
+main(){
+    setup
+
+    cd ${BASE_DIR}
+
+    ansible-playbook -i inventory/prow-performance-workloads/hosts.yaml  --become --become-user=root --private-key ~/.ssh/id_ed25519 bootstrap.yml
```

Comment:
I think the name under the `inventory` dir should match the top-level dir, IMO we should either name both `prow-sig-scale-workloads` or `prow-performance-workloads`.

Reply:
I started with `sig-scale-workloads`, then changed to `prow-performance-workloads`
It is a left over, I will update it